site/server/app/com/fortysevendeg/exercises/controllers/ApplicationController.scala

```diff
@@ -35,8 +40,8 @@ class ApplicationController(userService: UserServices)(implicit exerciseOps: Exe
     exerciseOps.getLibraries.runTask match {
       case \/-(libraries) ⇒
         request.session.get("oauth-token") map { token ⇒
-          val user = userService.getUserByLogin(GetUserByLoginRequest(login = request.session.get("user").getOrElse("")))
-          user.map(response ⇒ Ok(views.html.templates.home.index(user = response.user, libraries = libraries)))
+          val user = userService.getUserByLogin(request.session.get("user").getOrElse(""))
```

Comment:
Not saying we should do any of this now but if the UserService is not already lifting to Free or using Free Algebras we should refactor it or have a ticket to do so.